app/views/boards/show.html.erb

```diff
@@ -52,7 +52,7 @@ See doc/COPYRIGHT.rdoc for more details.
       <%= preview_link preview_board_topics_path(@board), 'message-form-preview', class: 'button' %>
       <%= link_to l(:button_cancel), "#", onclick: 'Element.hide("add-message")', class: 'button' %>
     <% end %>
-    <div id="preview" class="wiki"></div>
+    <div id="preview"></div>
```

Comment:
@NobodysNightmare I actually discussed doing further refactoring here with @floriank . See: 
* https://github.com/opf/openproject/pull/2502#issuecomment-73216414
* https://gist.github.com/myabc/7506ad8b27eb35a6a019

Maybe in a future release… :disappointed: 

We should also wait for some design feedback (see https://github.com/opf/openproject/pull/2815#issuecomment-92356117)